.buildkite/scripts/common.sh

```diff
@@ -109,11 +120,32 @@ with_go() {
   export PATH="${go_path}:${PATH}"
 }
 
+checkLinuxType() {
+  if [ "${platform_type}" == "Linux" ]; then
+    if grep -q 'ubuntu' /etc/*release; then
```

Comment:
All Linux distros have a file `/etc/os-release` so we can be more specific here (see https://github.com/chef/os_release)

Reply:
changed